resources/views/list-view/sorting.blade.php

```diff
@@ -0,0 +1,36 @@
+{{-- list-view.sorting.blade
+
+Renders the dropdown for the sorting select
+To customize it you should shange the UI component used, YOU SHOULD NOT CUSTOMIZE ANYHING HERE
+UI components used:
+  - form/select --}}
+<div class="leading-tight">
+  {{-- Sorting dropdown --}}
+  @component('laravel-views::components.drop-down')
+    @slot('trigger')
+      @include('laravel-views::components.buttons.select', [
+      'title'=> $sortableBy->flip()->get($sortBy) ?? __('Sort By'),
+      ])
+    @endslot
+    {{-- Each sortable item --}}
+    <div class="p-2 text-sm">
+      {{ __('Sort By') }}
+    </div>
+    @foreach ($sortableBy as $title => $column)
+      <div class="hover:bg-gray-200">
```

Comment:
Looks good, either way, you could use the same styles as the `actions.icon-and-title` components, at the end both of them are items (submenu) in a dropdown